docker-compose.yml

```diff
@@ -436,12 +436,7 @@ services:
       - shared-database
 
   probate-back-office:
-    build:
-      context: .
-    dns:
-      - 172.16.0.10
-      - 172.16.0.14
-      - 8.8.8.8
+    image: hmctspublic.azurecr.io/probate/back-office:pr-1198
```

Comment:
I believe this one you would change before pushing the changes to master

Reply:
Yes of course - all the probate specific images should be reverted to latest once the code is live